common/views/components/SearchForm/SearchForm.tsx

```diff
@@ -219,7 +219,6 @@ const SearchForm = forwardRef(
           <SearchFilters
             query={query}
             linkResolver={linkResolver}
-            searchForm={searchForm}
```

Comment:
So `searchForm` is a `ref` or a `string` depending on where it's used in the site, is that right?

Reply:
no, it was never used as a ref, because in other locations, this component was the child of a form, and passing a ref was not needed.
So this component and its children never made use of that functionality until now. And now, it is being used with the native html feature.

Reply:
I think I got confused with [its use in the current form](https://github.com/wellcomecollection/wellcomecollection.org/blob/85d4e78f47d2871a012c3f127d0923b7e1f776c1/common/views/components/SearchForm/SearchForm.tsx#L92)

Reply:
that is actually a good point, should I use a different name so that there is less confusion? Or?

Reply:
I'll go for `formId` for another layer of clarity